### DISCUSSION TITLE
RISC-V GCC Toolchains rv32i-4.0.0安裝

### DISCUSSION THREAD

**Discussion by ghost** [bol-edu/HLS-SOC-Discussions#129] [General]
想請教一下，我照著工具書上打指令，但都一直顯示無法成功，想請問大家需要怎麼解決

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/119398292/05573e8f-92f4-4d37-930e-64a01e9030c6)

**Comment by LingChiYang** [comment]
$ export PATH=$PATH:/opt/riscv/bin
